kolibri/core/assets/src/utils/vue-intl-locale-data.js

```diff
@@ -8,6 +8,7 @@
  */
 module.exports = function() {
   const data = [];
+  data.push(require('vue-intl/locale-data/ach.js'));
```

Comment:
Likewise here, vue-intl will not have any data for this Crowdin made up language :)